pytorch_lightning/trainer/trainer.py

```diff
@@ -119,7 +118,7 @@ def __init__(
             distributed_backend: Optional[str] = None,
             precision: int = 32,
             print_nan_grads: bool = False,  # backward compatible, todo: remove in v0.9.0
-            weights_summary: Optional[str] = 'full',
+            weights_summary: Optional[str] = 'top',
```

Comment:
This change is not reflected in the docs

Reply:
mind shot a fix PR?

Reply:
yes unless it was not intentional. @williamFalcon yes or no?

Reply:
sorry, forgot to update docs on this. 
i propose we make it top going forward

Reply:
Added it here #2021, hope it's fine including it there.